### ISSUE TITLE
stress: failed test in cockroach/cli/cli.test: Example_sql_escape

### ISSUE THREAD

**Issue by jordanlewis** [cockroachdb/cockroach#7907]
Binary: cockroach/static-tests.tar.gz sha: https://github.com/cockroachdb/cockroach/commits/1939baa458fd4d2d817b8befdbc40e24ffa88472

Stress build found a failed test:

```
=== RUN   Example_sql_escape
I160719 05:30:51.108245 storage/engine/rocksdb.go:299  opening in memory rocksdb instance
W160719 05:30:51.111226 gossip/gossip.go:937  not connected to cluster; use --join to specify a connected node
W160719 05:30:51.112179 gossip/gossip.go:937  not connected to cluster; use --join to specify a connected node
I160719 05:30:51.112215 server/node.go:401  store store=0:0 ([]=) not bootstrapped
I160719 05:30:51.114075 storage/replica_command.go:1642  store=1:1 ([]=) range=1 [/Min-/Max): new range lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC 408029h30m55.613624699s following replica {0 0 0} 1970-01-01 00:00:00 +0000 UTC 0 [physicalTime=2016-07-19 05:30:51.11398437 +0000 UTC]
I160719 05:30:51.114827 server/node.go:327  **** cluster {7b515894-4384-45bf-8f5a-5f1815311acd} has been created
I160719 05:30:51.114851 server/node.go:328  **** add additional nodes by specifying --join=127.0.0.1:40139
W160719 05:30:51.114861 gossip/gossip.go:937  not connected to cluster; use --join to specify a connected node
I160719 05:30:51.115871 server/node.go:414  initialized store store=1:1 ([]=): {Capacity:10365558784 Available:8154660864 RangeCount:0}
I160719 05:30:51.117026 server/node.go:302  node ID 1 initialized
I160719 05:30:51.117071 storage/stores.go:298  read 0 node addresses from persistent storage
I160719 05:30:51.117109 server/node.go:537  connecting to gossip network to verify cluster ID...
I160719 05:30:51.117126 server/node.go:558  node connected via gossip and verified as part of cluster {"7b515894-4384-45bf-8f5a-5f1815311acd"}
I160719 05:30:51.117146 server/node.go:355  [node=1] Started node with [[]=] engine(s) and attributes []
I160719 05:30:51.117162 server/server.go:425  starting https server at 127.0.0.1:41836
I160719 05:30:51.117171 server/server.go:426  starting grpc/postgres server at 127.0.0.1:40139
I160719 05:30:51.124077 storage/split_queue.go:100  splitting store=1:1 ([]=) range=1 [/Min-/Max) at keys [/Table/11/0 /Table/12/0 /Table/13/0 /Table/14/0]
I160719 05:30:51.124228 sql/event_log.go:103  Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:40139} Attrs:} ClusterID:{UUID:7b515894-4384-45bf-8f5a-5f1815311acd} StartedAt:1468906251117132450}
I160719 05:30:51.141787 sql/event_log.go:103  Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:40139} Attrs:} ClusterID:{UUID:7b515894-4384-45bf-8f5a-5f1815311acd} StartedAt:1468906251117132450}
I160719 05:30:51.141823 storage/replica_command.go:2215  store=1:1 ([]=) range=1 [/Min-/Max): initiating a split of this range at key /Table/11
I160719 05:30:51.144459 storage/log.go:84  Range Event: "split", range: 1, info: {"UpdatedDesc":{"range_id":1,"end_key":"kw==","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2},"NewDesc":{"range_id":2,"start_key":"kw==","end_key":"//8=","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2}}
I160719 05:30:51.147736 storage/log.go:84  Range Event: "split", range: 1, info: {"UpdatedDesc":{"range_id":1,"end_key":"kw==","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2},"NewDesc":{"range_id":2,"start_key":"kw==","end_key":"//8=","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2}}
I160719 05:30:51.151966 storage/replica_command.go:1642  store=1:1 ([]=) range=2 [/Table/11-/Max): new range lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC 408029h30m55.715224982s following replica {0 0 0} 1970-01-01 00:00:00 +0000 UTC 0 [physicalTime=2016-07-19 05:30:51.151865485 +0000 UTC]
I160719 05:30:51.152118 storage/replica_command.go:2215  store=1:1 ([]=) range=2 [/Table/11-/Max): initiating a split of this range at key /Table/12
I160719 05:30:51.153038 storage/log.go:84  Range Event: "split", range: 2, info: {"UpdatedDesc":{"range_id":2,"start_key":"kw==","end_key":"lA==","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2},"NewDesc":{"range_id":3,"start_key":"lA==","end_key":"//8=","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2}}
I160719 05:30:51.156723 storage/replica_command.go:1642  store=1:1 ([]=) range=3 [/Table/12-/Max): new range lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC 408029h30m55.715224982s following replica {0 0 0} 1970-01-01 00:00:00 +0000 UTC 0 [physicalTime=2016-07-19 05:30:51.156622175 +0000 UTC]
I160719 05:30:51.156880 storage/replica_command.go:2215  store=1:1 ([]=) range=3 [/Table/12-/Max): initiating a split of this range at key /Table/13
I160719 05:30:51.158025 storage/log.go:84  Range Event: "split", range: 3, info: {"UpdatedDesc":{"range_id":3,"start_key":"lA==","end_key":"lQ==","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2},"NewDesc":{"range_id":4,"start_key":"lQ==","end_key":"//8=","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2}}
I160719 05:30:51.161450 storage/replica_command.go:1642  store=1:1 ([]=) range=4 [/Table/13-/Max): new range lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC 408029h30m55.715224982s following replica {0 0 0} 1970-01-01 00:00:00 +0000 UTC 0 [physicalTime=2016-07-19 05:30:51.16135335 +0000 UTC]
I160719 05:30:51.161590 storage/replica_command.go:2215  store=1:1 ([]=) range=4 [/Table/13-/Max): initiating a split of this range at key /Table/14
I160719 05:30:51.162652 storage/log.go:84  Range Event: "split", range: 4, info: {"UpdatedDesc":{"range_id":4,"start_key":"lQ==","end_key":"lg==","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2},"NewDesc":{"range_id":5,"start_key":"lg==","end_key":"//8=","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2}}
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e create database t; create table t.t (s string, d string);]
Error: remote error: bad record MAC
E160719 05:30:51.246000 server/server.go:371  local error: bad record MAC
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e insert into t.t values (e'foo', 'printable ASCII')]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e insert into t.t values (e'"foo', 'printable ASCII with quotes')]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e insert into t.t values (e'\\foo', 'printable ASCII with backslash')]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e insert into t.t values (e'foo\x0abar', 'non-printable ASCII')]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e insert into t.t values ('κόσμε', 'printable UTF8')]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e insert into t.t values (e'\xc3\xb1', 'printable UTF8 using escapes')]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e insert into t.t values (e'\x01', 'non-printable UTF8 string')]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e insert into t.t values (e'\xdc\x88\x38\x35', 'UTF8 string with RTL char')]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e insert into t.t values (e'\xc3\x28', 'non-UTF8 string')]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e insert into t.t values (e'a\tb\tc\n12\t123123213\t12313', 'tabs')]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e select * from t.t]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e create table t.u ("""foo" int, "\foo" int, "foo
bar" int, "κόσμε" int, "܈85" int)]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e insert into t.u values (0, 0, 0, 0, 0)]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e show columns from t.u]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key -e select * from t.u]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key --pretty -e select * from t.t]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key --pretty -e show columns from t.u]
Error: pq: database "t" does not exist
[sql --host=127.0.0.1 --port=40139 --insecure=false --ca-cert=/tmp/cli-test106389613/ca.crt --cert=/tmp/cli-test106389613/node.crt --key=/tmp/cli-test106389613/node.key --pretty -e select * from t.u]
Error: pq: database "t" does not exist
--- FAIL: Example_sql_escape (0.53s)
got:
sql -e create database t; create table t.t (s string, d string);
remote error: bad record MAC
sql -e insert into t.t values (e'foo', 'printable ASCII')
pq: database "t" does not exist
sql -e insert into t.t values (e'"foo', 'printable ASCII with quotes')
pq: database "t" does not exist
sql -e insert into t.t values (e'\\foo', 'printable ASCII with backslash')
pq: database "t" does not exist
sql -e insert into t.t values (e'foo\x0abar', 'non-printable ASCII')
pq: database "t" does not exist
sql -e insert into t.t values ('κόσμε', 'printable UTF8')
pq: database "t" does not exist
sql -e insert into t.t values (e'\xc3\xb1', 'printable UTF8 using escapes')
pq: database "t" does not exist
sql -e insert into t.t values (e'\x01', 'non-printable UTF8 string')
pq: database "t" does not exist
sql -e insert into t.t values (e'\xdc\x88\x38\x35', 'UTF8 string with RTL char')
pq: database "t" does not exist
sql -e insert into t.t values (e'\xc3\x28', 'non-UTF8 string')
pq: database "t" does not exist
sql -e insert into t.t values (e'a\tb\tc\n12\t123123213\t12313', 'tabs')
pq: database "t" does not exist
sql -e select * from t.t
pq: database "t" does not exist
sql -e create table t.u ("""foo" int, "\foo" int, "foo
bar" int, "κόσμε" int, "܈85" int)
pq: database "t" does not exist
sql -e insert into t.u values (0, 0, 0, 0, 0)
pq: database "t" does not exist
sql -e show columns from t.u
pq: database "t" does not exist
sql -e select * from t.u
pq: database "t" does not exist
sql --pretty -e select * from t.t
pq: database "t" does not exist
sql --pretty -e show columns from t.u
pq: database "t" does not exist
sql --pretty -e select * from t.u
pq: database "t" does not exist
want:
sql -e create database t; create table t.t (s string, d string);
CREATE TABLE
sql -e insert into t.t values (e'foo', 'printable ASCII')
INSERT 1
sql -e insert into t.t values (e'"foo', 'printable ASCII with quotes')
INSERT 1
sql -e insert into t.t values (e'\\foo', 'printable ASCII with backslash')
INSERT 1
sql -e insert into t.t values (e'foo\x0abar', 'non-printable ASCII')
INSERT 1
sql -e insert into t.t values ('κόσμε', 'printable UTF8')
INSERT 1
sql -e insert into t.t values (e'\xc3\xb1', 'printable UTF8 using escapes')
INSERT 1
sql -e insert into t.t values (e'\x01', 'non-printable UTF8 string')
INSERT 1
sql -e insert into t.t values (e'\xdc\x88\x38\x35', 'UTF8 string with RTL char')
INSERT 1
sql -e insert into t.t values (e'\xc3\x28', 'non-UTF8 string')
INSERT 1
sql -e insert into t.t values (e'a\tb\tc\n12\t123123213\t12313', 'tabs')
INSERT 1
sql -e select * from t.t
10 rows
s	d
foo	printable ASCII
"\"foo"	printable ASCII with quotes
"\\foo"	printable ASCII with backslash
"foo\nbar"	non-printable ASCII
"\u03ba\u1f79\u03c3\u03bc\u03b5"	printable UTF8
"\u00f1"	printable UTF8 using escapes
"\x01"	non-printable UTF8 string
"\u070885"	UTF8 string with RTL char
"\xc3("	non-UTF8 string
"a\tb\tc\n12\t123123213\t12313"	tabs
sql -e create table t.u ("""foo" int, "\foo" int, "foo
bar" int, "κόσμε" int, "܈85" int)
CREATE TABLE
sql -e insert into t.u values (0, 0, 0, 0, 0)
INSERT 1
sql -e show columns from t.u
6 rows
Field	Type	Null	Default
"\"foo"	INT	true	NULL
"\\foo"	INT	true	NULL
"foo\nbar"	INT	true	NULL
"\u03ba\u1f79\u03c3\u03bc\u03b5"	INT	true	NULL
"\u070885"	INT	true	NULL
rowid	INT	false	unique_rowid()
sql -e select * from t.u
1 row
"\"foo"	"\\foo"	"foo\nbar"	"\u03ba\u1f79\u03c3\u03bc\u03b5"	"\u070885"
0	0	0	0	0
sql --pretty -e select * from t.t
+--------------------------------+--------------------------------+
|               s                |               d                |
+--------------------------------+--------------------------------+
| foo                            | printable ASCII                |
| "foo                           | printable ASCII with quotes    |
| \foo                           | printable ASCII with backslash |
| foo␤                           | non-printable ASCII            |
| bar                            |                                |
| κόσμε                          | printable UTF8                 |
| ñ                              | printable UTF8 using escapes   |
| "\x01"                         | non-printable UTF8 string      |
| ܈85                            | UTF8 string with RTL char      |
| "\xc3("                        | non-UTF8 string                |
| a   b         c␤               | tabs                           |
| 12  123123213 12313            |                                |
+--------------------------------+--------------------------------+
(10 rows)
sql --pretty -e show columns from t.u
+----------+------+-------+----------------+
|  Field   | Type | Null  |    Default     |
+----------+------+-------+----------------+
| "foo     | INT  | true  | NULL           |
| \foo     | INT  | true  | NULL           |
| foo␤     | INT  | true  | NULL           |
| bar      |      |       |                |
| κόσμε    | INT  | true  | NULL           |
| ܈85      | INT  | true  | NULL           |
| rowid    | INT  | false | unique_rowid() |
+----------+------+-------+----------------+
(6 rows)
sql --pretty -e select * from t.u
+------+------+------------+-------+-----+
| "foo | \foo | "foo\nbar" | κόσμε | ܈85 |
+------+------+------------+-------+-----+
|    0 |    0 |          0 |     0 |   0 |
+------+------+------------+-------+-----+
(1 row)
```

Run Details:

```
0 runs so far, 0 failures, over 5s
0 runs so far, 0 failures, over 10s
16 runs so far, 0 failures, over 15s
16 runs so far, 0 failures, over 20s
17 runs so far, 0 failures, over 25s
32 runs so far, 0 failures, over 30s
32 runs so far, 0 failures, over 35s
48 runs so far, 0 failures, over 40s
48 runs so far, 0 failures, over 45s
50 runs so far, 0 failures, over 50s
64 runs so far, 0 failures, over 55s
64 runs so far, 0 failures, over 1m0s
80 runs so far, 0 failures, over 1m5s
80 runs so far, 0 failures, over 1m10s
82 runs so far, 0 failures, over 1m15s
96 runs so far, 0 failures, over 1m20s
97 runs so far, 0 failures, over 1m25s
112 runs so far, 0 failures, over 1m30s
113 runs so far, 0 failures, over 1m35s
116 runs so far, 0 failures, over 1m40s
129 runs so far, 0 failures, over 1m45s
130 runs so far, 0 failures, over 1m50s
144 runs so far, 0 failures, over 1m55s
146 runs so far, 0 failures, over 2m0s
150 runs so far, 0 failures, over 2m5s
162 runs so far, 0 failures, over 2m10s
164 runs so far, 0 failures, over 2m15s
178 runs so far, 0 failures, over 2m20s
179 runs so far, 0 failures, over 2m25s
185 runs so far, 0 failures, over 2m30s
195 runs so far, 0 failures, over 2m35s
197 runs so far, 0 failures, over 2m40s
211 runs so far, 0 failures, over 2m45s
213 runs so far, 0 failures, over 2m50s
222 runs so far, 0 failures, over 2m55s
229 runs so far, 0 failures, over 3m0s
231 runs so far, 0 failures, over 3m5s
245 runs so far, 0 failures, over 3m10s
246 runs so far, 0 failures, over 3m15s
257 runs so far, 0 failures, over 3m20s
262 runs so far, 0 failures, over 3m25s
264 runs so far, 0 failures, over 3m30s
278 runs so far, 0 failures, over 3m35s
279 runs so far, 0 failures, over 3m40s
293 runs so far, 0 failures, over 3m45s
295 runs so far, 0 failures, over 3m50s
297 runs so far, 0 failures, over 3m55s
311 runs so far, 0 failures, over 4m0s
313 runs so far, 0 failures, over 4m5s
326 runs so far, 0 failures, over 4m10s
329 runs so far, 0 failures, over 4m15s
331 runs so far, 0 failures, over 4m20s
345 runs so far, 0 failures, over 4m25s
345 runs so far, 0 failures, over 4m30s
360 runs so far, 0 failures, over 4m35s
361 runs so far, 0 failures, over 4m40s
366 runs so far, 0 failures, over 4m45s
377 runs so far, 0 failures, over 4m50s
378 runs so far, 0 failures, over 4m55s
393 runs so far, 0 failures, over 5m0s
394 runs so far, 0 failures, over 5m5s
398 runs so far, 0 failures, over 5m10s
410 runs so far, 0 failures, over 5m15s
411 runs so far, 0 failures, over 5m20s
426 runs so far, 0 failures, over 5m25s
426 runs so far, 0 failures, over 5m30s
430 runs so far, 0 failures, over 5m35s
442 runs so far, 0 failures, over 5m40s
445 runs so far, 0 failures, over 5m45s
458 runs so far, 0 failures, over 5m50s
459 runs so far, 0 failures, over 5m55s
468 runs so far, 0 failures, over 6m0s
475 runs so far, 0 failures, over 6m5s
478 runs so far, 0 failures, over 6m10s
491 runs so far, 0 failures, over 6m15s
493 runs so far, 0 failures, over 6m20s
505 runs so far, 0 failures, over 6m25s
510 runs so far, 0 failures, over 6m30s
510 runs so far, 0 failures, over 6m35s
526 runs so far, 0 failures, over 6m40s
526 runs so far, 0 failures, over 6m45s
537 runs so far, 0 failures, over 6m50s
542 runs so far, 0 failures, over 6m55s
545 runs so far, 0 failures, over 7m0s
558 runs so far, 0 failures, over 7m5s
560 runs so far, 0 failures, over 7m10s
573 runs so far, 0 failures, over 7m15s
574 runs so far, 0 failures, over 7m20s
578 runs so far, 0 failures, over 7m25s
590 runs so far, 0 failures, over 7m30s
593 runs so far, 0 failures, over 7m35s
606 runs so far, 0 failures, over 7m40s
609 runs so far, 0 failures, over 7m45s
612 runs so far, 0 failures, over 7m50s
625 runs so far, 0 failures, over 7m55s
627 runs so far, 0 failures, over 8m0s
640 runs so far, 0 failures, over 8m5s
643 runs so far, 0 failures, over 8m10s
646 runs so far, 0 failures, over 8m15s
659 runs so far, 0 failures, over 8m20s
660 runs so far, 0 failures, over 8m25s
673 runs so far, 0 failures, over 8m30s
676 runs so far, 0 failures, over 8m35s
682 runs so far, 0 failures, over 8m40s
692 runs so far, 0 failures, over 8m45s
693 runs so far, 0 failures, over 8m50s
708 runs so far, 0 failures, over 8m55s
708 runs so far, 0 failures, over 9m0s
716 runs so far, 0 failures, over 9m5s
724 runs so far, 0 failures, over 9m10s
726 runs so far, 0 failures, over 9m15s
740 runs so far, 0 failures, over 9m20s
741 runs so far, 0 failures, over 9m25s
756 runs so far, 0 failures, over 9m30s
757 runs so far, 0 failures, over 9m35s
761 runs so far, 0 failures, over 9m40s
773 runs so far, 0 failures, over 9m45s
774 runs so far, 0 failures, over 9m50s
789 runs so far, 0 failures, over 9m55s
790 runs so far, 0 failures, over 10m0s
795 runs so far, 0 failures, over 10m5s
806 runs so far, 0 failures, over 10m10s
809 runs so far, 0 failures, over 10m15s
816 runs completed, 1 failures, over 10m18s
FAIL
```
Please assign, take a look and update the issue accordingly.

**Comment by RaduBerinde** [comment]
Hm, `remote error: bad record MAC` is a TLS error. Couldn't repro locally (with `make stress PKG=./cli TESTS=Example_sql_escape`).

**Comment by tamird** [comment]
There were TLS-related fixed we landed in the 1.7 cycle. This might go away when we move up to 1.7.

**Comment by bdarnell** [comment]
Bugs that would lead to "bad record MAC" would probably warrant a security fix in the 1.6.x series instead of waiting for 1.7. So I wouldn't bet on this going away with the upgrade, although if it doesn't reproduce under stress I don't have any better idea about where to look for it.

**Comment by tamird** [comment]
Closing for now due to lack of repro.
